en/developer_guide/field_type.rst

```diff
@@ -0,0 +1,107 @@
+Field type
+==========
+
+Open Orchestra offers the possibility to create the Content Types from the back office.
+Further information about it is available in the `Content Types documentation`_.
+
+When creating a content type, you can add several attributes of different types.
+By default Open Orchestra offers few types
+(text line, date, email, hidden, integer, choice, media, money, rich test, text area).
+
+.. image:: ../../images/fieldType.png
+
+Add a field type
+----------------
+To add a new field type, you can add an entry to the configuration ``open_orchestra_backoffice.field_types``
+which describes your field type.
+
+The description of your field type must contain these values:
+
+* ``name`` name of your field type
+* ``type`` form type of your field, this can be a form type natively available in Symfony or a custom form type
+* ``label`` translation key of label
+* ``default_value`` (optional)
+* ``options`` (optional)
+
+``default_value`` describes the field allowing to contribute the default value of your field.
+
+``options`` describes the different options to configure your field type (max length, required, etc).
+
+Example of simple field type:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.field_types:
+        custom_textarea:
+            label: open_orchestra.form.field_type.textarea
+            type: textarea
+
+Field type options
+------------------
+
+The field type can be configure with options, for instance to add an option
+to field ``custom_textarea`` to configure the max length.
+
+You must add an entry in configuration ``open_orchestra_backoffice.options`` with 4 parameters:
+
+* ``name`` name of option
+* ``type`` form type of option, this can be a form type natively available in Symfony or your custom form type
+* ``label`` translation key of label
+* ``required`` a boolean indicating if option is required
+
+When the user adds a field type at a content type, if an option of this field type is required then
+the user can't save the content type until he has contributed the option.
+
+Example of option ``max length``:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.options:
+        max_length:
+            type: integer
+            label: open_orchestra_backoffice.form.orchestra_fields.max_length
+            required: true
+
+Now, we can add the option in your field type:
```

Comment:
you mix "we" and "your": "we can add the option in our field" or "you can add the option in your field"